File: PHP/Figures exercise/figures/triangle.php
<?php
 class Triangle extends FigureGeometric {

    private float $base;
    private float $height;
    private float $calculate;
    
    public function __construct(float $base, float $height){
        parent::_construct("Triangle");
        $this-> base= $base;
        $this->height=$height;
    }
    
    public function getBase():float{
        return $this-> base;
    }
    public function getHeight():float{
        return $this-> height;
    }
    public function calculateArea():float{
        $this->calculate = ($this-> getBase()* $this->getHeight())/2;
        return $this->calculate;
    }
}

File: JAVA/Figures exercise/com/figures/Triangle.java
package com.figures;

public class Triangle extends FigureGeometric {
    private int base;
    private int height;
    private double calculate;

    public  Triangle (int base, int height) {
        super("Triangle", "Triangulo");
        this.base = base;
        this.height= height;
    }
    public int getBase() {
        return this.base;
    }
    public int getHeight(){
        return this.height;
    }
    public double calculateArea() {
        this.calculate = (this.getBase() * this.getHeight())/2;
        return this.calculate;
    }
    
}